src/libraries/Directory.Build.targets

```diff
@@ -272,4 +273,20 @@
       </ItemGroup>
     </When>
   </Choose>
+
+  <Target Name="AddLibIntelDocs">
```

Comment:
Cool this will make it very simple to hook the hybrid mode for docs in between intellisense and triple slash.

Also, could you rename to LibIntellisenseDocs? That's more clear.